src/en/charms-storage.md

```diff
@@ -466,9 +466,9 @@ provider currently supports a single pool configuration attribute:
 ### LXD (lxd)
 
 !!! Note:
-    To get an LXD version (at least 2.16) on either Ubuntu 14.04 LTS (Trusty)
-    or Ubuntu 16.04 LTS (Xenial) that has the 'lxd' storage provider feature
-    the [LXD:stable PPA][ppa-lxd] will be required.
+    Ubuntu 14.04 LTS (Trusty) and Ubuntu 16.04 LTS (Xenial) do not ship with a
+    version of LXD (at least 2.16) that has the 'lxd' storage provider feature.
```

Comment:
I found the version number slightly confusing on first read, like it's the version found in those old distros. I'd move *(at least 2.16)* after the explanation that you need LXD with a storage provider and take it out of brackets, such as *,available in LXD 2.16 and later.*

Reply:
:+1: